tests/test_helpers.py

```diff
@@ -0,0 +1,96 @@
+"""Tests for test helpers."""
+
+from uuid import uuid4
+
+import pytest
+
+from .helpers import (
+    assert_statement_get_responses_are_equivalent,
+    string_is_date,
+    string_is_uuid,
+)
+
+
+def test_utils_string_is_date():
+    """Test that strings representing dates are properly identified."""
+    string = "2022-06-22T08:31:38+00:00"
+    assert string_is_date(string)
+
+    string = "a2023-06-22T08:31:38+00:00"
+    assert not string_is_date(string)
+
+
+def test_utils_string_is_uuid():
+    """Test that strings representing uuids are properly identified."""
+    string = str(uuid4())
+    assert string_is_uuid(string)
+
+    string = "not_a_valid_uuid"
+    assert not string_is_uuid(string)
+
+
+@pytest.mark.parametrize(
+    "modified_fields,are_equivalent",
+    [
+        (
+            {
+                "timestamp": None,
+                "version": None,
+                "authority": "authority_2",
+                "stored": "stored_2",
+            },
+            True,
+        ),
+        ({"actor": {"actor_field": "actor_2"}}, False),
+        ({"verb": "verb_2"}, False),
+        ({"object": "object_2"}, False),
+        ({"id": "id_2"}, False),
+        ({"result": "result_2"}, False),
+        ({"context": "context_2"}, False),
+        ({"attachements": "attachements_2"}, False),
+        ({"timestamp": "timestamp_2"}, False),
+        ({"version": "version_2"}, False),
+    ],
+)
+def test_utils_assert_statement_get_responses_are_equivalent(
```

Comment:
What about a dedicated test for the `statements_are_equivalent` function in `test_utils.py`?

Reply:
I would avoid the double testing for duplication as the behavior is tested here. But I did add a condition (using `match`) to make sure that `statements_are_equivalent` is actually being tested which was not the case previously:

https://github.com/openfun/ralph/compare/feature/enrich-incoming-statements#diff-08a6944653efc734f4c8cb856fc854b9f21d655ca69796af2b170c53bff7919eR90